account_invoice_refund_reason/README.rst

```diff
@@ -0,0 +1,32 @@
+Overview
+========
+
+This module allows you to define a list of reasons to create a credit note from an invoice and report on them.
+
+Configuration
+=============
+Go to Accounting > Configuration > Management > Refund Reasons
+Create the different reasons you want to provide users
+
+Usage
+=====
+
+Operations
+==========
+
+Go to Accounting > Customers > Invoices (or Accounting > Vendors > Bills)
+Select an open invoice (or bill)
+Click on the "Add Credit Note"
+Select the reason and create the credit note
+
+Credits
+=======
+
+* Open Source Integrators <http://www.opensourceintegrators.com>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
+
+Contributors
+============
+
+* Open Source Integrators <http://www.opensourceintegrators.com>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
```

Comment:
@Chanakya-SerpentCS Please use the template available at https://github.com/OCA/maintainer-tools/tree/master/template/module/